packages/mainmenu-extension/src/index.ts

```diff
@@ -92,6 +92,8 @@ export namespace CommandIDs {
 
   export const runAll = 'runmenu:run-all';
 
+  export const runAllMarkdown = 'runmenu:run-all-markdown';
```

Comment:
I don't think this ID should be here at all -- there is no `'runmenu:run-all-markdown'` command defined as far as I see.